pyproject.toml

```
@@ -1,28 +1,32 @@
-[tool.poetry]
+[project]
 name = "beng-lod-server"
-version = "1.0"
 description = "Webserver serving LOD for the NISV catalogue items, datasets and GTAA."
-authors = [
-  "Willem Melder <wmelder@beeldengeluid.nl>",
-  "Mari Wigham <mwigham@beeldengeluid.nl>",
-  "Govert Brinkmann <gbrinkmann@beeldengeluid.nl>"
-  ]
-license = "MIT"
+version = "2.1.2"
```

```suggestion
version = "1.0"
```

[`project.version`](https://python-poetry.org/docs/pyproject/#version) concerns the package, not poetry. for the later you can use [`tool.poetry.requires-poetry`](https://python-poetry.org/docs/pyproject/#version) voor